gini/gini-vision-lib-android
# eps e-payment QR Code url extraction

When an eps e-payment QR Code is detected, the contents of the QR Code is returned as an extraction to the client without sending it to the Gini API. The extraction name is `epsPaymentQRCodeUrl`.

### How to test
Run one of the example apps and scan an eps QR Code. To generate a QR Code go [here](https://eservice.stuzza.at/en/kundenservice/eps-demoshop/eps-demoshop-zentrale-bankauswahl.html), click on `eps-Überweisung`, enter `BKAUATWW` then click `Weiter zum Online-Banking` and finally click `Mit MobileBanking App fortfahren` to see the QR Code.
